app/controllers/ems_cloud_controller.rb

```diff
@@ -160,7 +160,7 @@ def ems_cloud_form_fields
                      :provider_id                     => @ems.provider_id ? @ems.provider_id : "",
                      :hostname                        => @ems.hostname,
                      :api_port                        => @ems.port,
-                     :api_version                     => @ems.api_version,
+                     :api_version                     => @ems.api_version ? @ems.api_version : "v2",
```

Comment:
@Fryguy Yeah I agree, it should go back to this code when switching ems type, doesn't make sense to do the businesses logic in JS, for infra it will actually have to, since we have e.g. different selectboxes values for different EMS (unless we will preload all data for all EMSes. which again seems wrong). 

Though that is currently above my angular skills :-D So will this suffice as a bacportable fix? While UI team should do later refactoring of the angular form?